apps/sensenet/src/hooks/use-drawer-items.tsx

```diff
@@ -164,6 +138,8 @@ export const useDrawerItems = () => {
               path: (item as CustomContentDrawerItem).settings?.appPath || '',
             },
           })
+        case 'Settings':
+          return resolvePathParams({ path: PATHS.settings.appPath })
```

Comment:
You should give a value to submenu which will be the default view. In PermanentDrawer it set to stats by a hacky way (I also commented there).